misc/python/materialize/scalability/schema.py

```diff
@@ -0,0 +1,67 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from enum import Enum, auto
+from typing import Optional
+
+
+class Source(Enum):
+    TABLE = auto()
+
+
+class TransactionIsolation(Enum):
+    SERIALIZABLE = "serializable"
+    STRICT_SERIALIZABLE = "strict serializable"
+
+    def __str__(self) -> str:
+        return self.value
+
+
+class Schema:
+    def __init__(
+        self,
+        source: Source = Source.TABLE,
+        schema: str = "scalability",
+        index: bool = True,
```

Comment:
Rename to `create_index` to make it clear already at this point what it does? 😀

Reply:
Yes thanks, done.